lib/kitchen/directions/bake_chapter_key_concepts.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    # Bake directions for eoc key concepts
+    #
+    module BakeChapterKeyConcepts
+      def self.v1(chapter:, metadata_source:)
+        metadata_elements = metadata_source.children_to_keep.copy
+
+        fix_titles = chapter.pages.map do |page|
+          key_concepts = page.key_concepts
+          next if key_concepts.none?
+
+          key_concepts.search('h3').trash
+          key_concepts.map do |key_concept|
+            id = key_concept.id.split('fs-id')
+            key_concept.prepend(child:
+              <<~HTML
+                <a href="##{id[0]}0">
+                  <h3 data-type="document-title" id="#{id[0]}0">
+                    <span class="os-number">#{chapter.count_in(:book)}.#{page.count_in(:chapter) - 1}</span>
+                    <span class="os-divider"> </span>
+                    <span class="os-text" data-type="" itemprop="">#{page.title.text}</span>
+                  </h3>
+                </a>
+              HTML
+            )
+
+            new_key_concept = key_concept.cut
+
+            <<~HTML
+              <div class="os-section-area">
+                #{new_key_concept.paste}
+              </div>
+            HTML
+          end.join("\n")
+        end.join("\n")
```

Comment:
This way of doing things is ok and is how I started when I first wrote Kitchen.  But because it is a little hard to follow, I added Rails-like ERB processing so you can make a `blah.xhtml.erb` file with embedded ruby [like this one for BakeIndex](https://github.com/openstax/kitchen/blob/main/lib/kitchen/directions/bake_index/v1.xhtml.erb).  Makes the building more complex XHTML easier to read and understand.